admin/commands/storage/read_range_blocks.go

```diff
@@ -0,0 +1,39 @@
+package storage
+
+import (
+	"context"
+
+	"github.com/onflow/flow-go/admin"
+	"github.com/onflow/flow-go/admin/commands"
+	"github.com/onflow/flow-go/cmd/util/cmd/read-block-light"
+	"github.com/onflow/flow-go/storage"
+)
+
+var _ commands.AdminCommand = (*ReadRangeBlocksCommand)(nil)
+
+type ReadRangeBlocksCommand struct {
+	blocks storage.Blocks
+}
+
+func NewReadRangeBlocksCommand(blocks storage.Blocks) commands.AdminCommand {
+	return &ReadRangeBlocksCommand{
+		blocks: blocks,
+	}
+}
+
+func (c *ReadRangeBlocksCommand) Handler(ctx context.Context, req *admin.CommandRequest) (interface{}, error) {
+	data, err := parseHeightRangeRequestData(req)
```

Comment:
maybe rename to reqData or reqParam.  data can refer to anything.  same elsewhere